setup.cfg

```
@@ -4,6 +4,9 @@ license_file = LICENSE
 [flake8]
 max-line-length = 99
 
+[pylint.message-control]
+disable = C0330,R0801,R0913,R0904,R0903,R0902,W0511,C0116,C0115,W0703,R0914,R0914,C0302,R0912,R0915,R0401
```

```suggestion
disable = E0401
```

This disables the `import-error`, while leaving all the other warnings in tact ;)

---

Oops!